packages/react-components/react-progress/stories/ProgressBar/ProgressBarDescription.md

```diff
@@ -2,13 +2,6 @@ A Progress provides a visual representation of content being loaded or processed
 
 <!-- Don't allow prettier to collapse code block into single line -->
 <!-- prettier-ignore -->
-> **⚠️ Preview components are considered unstable:**
->
-> ```jsx
->
-> import { ProgressBar } from '@fluentui/react-components/unstable';
->
-> ```
->
-> - Features and APIs may change before final release
-> - Please contact us if you intend to use this in your product
```

Comment:
nit: I think this change should be done when moving the component to stable since for now it's still under unstable.